templates/CRM/Member/Form/PaymentPlanToggler.tpl

```diff
@@ -1,180 +1,210 @@
-          <div class="crm-section check_number-section">
-            <div class="label">Invoice Date</div>
-            <div class="content" id="invoice_date_summary"></div>
-            <div class="clear"></div>
-          </div>
-        </div>
-        <div class="crm-section billing_mode-section pay-later_info-section">
-          <div class="crm-section check_number-section">
-            <div class="label">Amount</div>
-            <div class="content" id="amount_summary"></div>
-            <div class="clear"></div>
-          </div>
-        </div>
-      </fieldset>
-    </td>
-  </tr>
-</table>
+<script type="text/javascript">
+  {literal}
+  if (typeof buildAutoRenew !== 'function') {
+    function buildAutoRenew () {}
+  }
+
+  (function ($) {
+    {/literal}
+    const togglerValue = '{$contribution_type_toggle}';
+    const membershipextrasAllMembershipData = {$allMembershipInfo};
+    const membershipextrasTaxRatesStr = '{$taxRates}';
+    const membershipextrasTaxTerm = '{$taxTerm}';
+    const membershipextrasCurrency = '{$currency}';
+    {literal}
+    const membershipextrasTaxRates = membershipextrasTaxRatesStr !== ''
+      ? JSON.parse(membershipextrasTaxRatesStr)
+      : [];
+
+    /**
+     * Perform changes on form to add payment plan as an option to pay for
+     * membership.
+     */
+    $(function () {
+      moveMembershipFormFields();
+      setMembershipFormEvents();
+      initializeMembershipForm();
+    });
+
+    /**
+     * Leaves form in a consistent state after the fields have been moved and the
+     * new events added.
+     */
+    function initializeMembershipForm () {
+      $('#invoice_date_summary').html($('#receive_date').val());
+      selectPaymentPlanTab(togglerValue);
+    }
+
+    /**
+     * Reorders fields within the form to follow a coherent logic with the new
+     * payment plan fields.
+     */
+    function moveMembershipFormFields () {
+      $('#contributionTypeToggle').insertBefore($('#is_different_contribution_contact').parent().parent());
+      $('#recordContribution legend:first').html('Contribution and Payment Plan');
+      $('#installments_row').insertAfter($('#financial_type_id').parent().parent());
+      $('#first_installment').insertAfter($('#installments_row'));
+      $('span.crm-error').css('display', 'none');
+      $('label span.crm-error').css('display', 'inline');
+      $('#payment_plan_fields_tabs').insertBefore($('#installments_row').closest('table'));
+    }
+
+    /**
+     * Selects Payment Plan tab and syncs the selection with radiobuttons
+     *
+     * @param {String} tabOptionId
+     */
+    function selectPaymentPlanTab (tabOptionId) {
+      const allTabsSelector = '#payment_plan_fields_tabs li';
+      const tabSelector = `${allTabsSelector}[data-selector=${tabOptionId}]`;
+
+      $(allTabsSelector).removeClass('ui-tabs-active');
+      $(tabSelector).addClass('ui-tabs-active');
```

Comment:
isn't this something that jQuery UI takes care of already? I think the class for this is `.ui-state-active`.

Reply:
Also, why not just click on the tab?

Reply:
![image](https://user-images.githubusercontent.com/3973243/53257314-b7197e00-36c1-11e9-8228-eb7adedc0c48.png)

jQuery UI is not available here...